{{cookiecutter.project_slug}}/setup.cfg

```diff
@@ -19,10 +19,3 @@ exclude = docs
 
 [aliases]
```

Comment:
I'd remove this section if it's empty

Reply:
I left this in to demonstrate how it's done, but I can certainly remove it